xpdacq/beamtime.py

```diff
@@ -461,7 +462,7 @@ def statTramp(dets, exposure, Tstart, Tstop, Tstep, sample_mapping, *,
 register_plan('Tramp', Tramp)
 register_plan('tseries', tseries)
 register_plan('Tlist', Tlist)
-register_plan('statTramp', statTramp)
+#register_plan('statTramp', statTramp)
```

Comment:
We might want a function (?) that loads up a bunch of plans (eg, "I'm running the cryostat so I need to load up all the cryostat runs"). This doesn't need to be in this PR.

Reply:
Yeah, that's good idea. I think it might make sense to put all those scripts in a centralized location or repo and give user the option to load/import more complicated function.

Create an issue at #462